app/screens/menu_action_selector/menu_action_selector.test.js

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import {shallow} from 'enzyme';
+import {IntlProvider} from 'react-intl';
+
+import MenuActionSelector from './menu_action_selector.js';
+
+jest.mock('rn-fetch-blob', () => ({
```

Comment:
We really need to make some standard mock functions for these at some point

Reply:
Right, I'll try to have it sooner.